### PR TITLE
Initial implementation of the bribri layout

### PR DESCRIPTION
**Pull request by elopio**
This is the layout for the Bribri language:
https://en.wikipedia.org/wiki/Bribri_language

Started from the spanish qwerty with symbols, and added the accents.

### CODE REVIEW

**Comment by elopio** [comment]
I followed the addons contributing guide, and copied things from other languages. But there are things that are still not clear. Please review and let me know what things are missing.

For example, this language has no flag. Should I use the Costa Rican flag?
Also finding a diccionary will be complicaded. I would like to do that in a follow up PR.

**Comment by menny** [comment]
This is amazing! Thank you so much for taking the time to create this pack.

PRs like these are the reason AnySoftKeyboard exists; the people create the packs they miss without the need for a _business case_.

**Comment by menny** [comment]
I'll review shortly. Costa Rican flag is okay here, or maybe just the letters *brib*? You decide.

File: addons/languages/bribri/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline addons/languages/bribri/apk/build.gradle:1]: The first publish is manual. Set this to `false` and add a reason (see https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/ae1a2ba7b05a200daf384127a5660f4b22c373a3/addons/languages/songhay/apk/build.gradle#L2)

While you at it, want to update the guides for adding language packs?

**Review by menny — CHANGES_REQUESTED** [review]

File: addons/languages/bribri/apk/build.gradle (hunk 1 above)
Comment (elopio) [reply addons/languages/bribri/apk/build.gradle]: done.

**Review by menny — DISMISSED** [review]

**Review by menny — APPROVED** [review]

**Comment by menny** [comment]
@elopio I pushed two commits to your branch (to fix an issue with the dictionary). Please pull from your branch

**Comment by menny** [comment]
Can you take a look at the failures? It could be that the filenames are incorrect

**Comment by menny** [comment]
I want to release all the language packs soon. Can we get this in before?

**Comment by menny** [comment]
I've created a new PR with your changes: https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/3170
I can either push my PR, or you can recreate the extra commits, from my PR, in yours

**Comment by menny** [comment]
I merged #3170, so we can close this PR.
Thank you for you contribution.

**Pull request closed without merging**